data/strings_str.dat

```diff
@@ -50,3 +50,9 @@ money_per_day = Money per day.
 location_modifiers = Bases at this location are %(modifiers)s.
 cpu_bonus = more efficient
 cpu_penalty = less efficient
+
+stat_cash_earned = Cash Earned
+stat_cpu_used = Cpu Used
```

Comment:
I would use CPU in all caps for consistency with (e.g.) the financial report screen.

Reply:
Absolutely.